example/package.json

```diff
@@ -19,8 +19,8 @@
     "test": "npm-run-all test:*"
   },
   "dependencies": {
-    "react": "15.4.1",
-    "react-native": "0.38.0",
+    "react": "~15.4.2",
+    "react-native": "0.40.0",
```

Comment:
As I know, this line is generated by `react-native init/upgrade` and I think example project should contain default code as much as possible. We can allow minor versions of react-native but I think it can "reduce stability" 🙂